src/nvim/ex_docmd.c

```diff
@@ -7858,19 +7858,21 @@ static void ex_stopinsert(exarg_T *eap)
  */
 void exec_normal_cmd(char_u *cmd, int remap, bool silent)
 {
+  // Stuff the argument into the typeahead buffer.
+  ins_typebuf(cmd, remap, 0, true, silent);
+  exec_normal(false);
+}
+
+void exec_normal(int was_typed)
```

Comment:
Missing comment from the original patch (might as well make it a docstring `///`)